pkg/ibmvpc/vpc.go

```diff
@@ -417,65 +417,65 @@ func (n *NACL) GeneralConnectivityPerSubnet(subnet *Subnet) string {
 	return res
 }
 
-func getNodeCidrs(n vpcmodel.Node) (subnetCidr, nodeCidr string, subnet *Subnet, err error) {
-	switch t := n.(type) {
+func subnetFromNode(node vpcmodel.Node) (subnet *Subnet, err error) {
+	switch concreteNode := node.(type) {
 	case *NetworkInterface:
-		return t.subnet.cidr, t.CidrOrAddress(), t.subnet, nil
+		return concreteNode.subnet, nil
 	case *IKSNode:
-		return t.subnet.cidr, t.CidrOrAddress(), t.subnet, nil
+		return concreteNode.subnet, nil
 	case *ReservedIP:
-		return t.subnet.cidr, t.CidrOrAddress(), t.subnet, nil
+		return concreteNode.subnet, nil
 	default:
-		return "", "", nil, fmt.Errorf("cannot get cidr for node: %+v", n)
+		return nil, fmt.Errorf("cannot get cidr for node: %+v", node)
```

Comment:
Correct the error message - the failure here is of getting `subnet` for node (when reviewing each commit separately you can't suggest changes)

Reply:
done